farm/modeling/prediction_head.py

```diff
@@ -18,6 +18,12 @@
 
 logger = logging.getLogger(__name__)
 
+try:
+    from apex.normalization.fused_layer_norm import FusedLayerNorm as BertLayerNorm
+except (ImportError, AttributeError) as e:
+    logger.info("Better speed can be achieved with apex installed from https://www.github.com/nvidia/apex .")
+    BertLayerNorm = torch.nn.LayerNorm
```

Comment:
Refer this https://github.com/huggingface/transformers/pull/1089

Reply:
:+1: